scripts/Makefile.connectal.build

```diff
@@ -362,12 +369,18 @@ $(DTOP)/bin/libconnectal-sim.so: $(SIM_OBJECTS) prepare_bin_target
 	$(Q)mkdir -p $(DTOP)/bin
 	$(CXX) -O -g -I$(DTOP)/jni -shared -fpic $(SIM_CXXFLAGS) -g -o $(DTOP)/bin/libconnectal-sim.so $(SIM_OBJECTS)
 
-bsim: prepare_bin_target $(DTOP)/bin/libconnectal-sim.so
```

Comment:
you should be using the verilog: target as an intermediate, like verilator and modelsim do rather than adding conditionals to the one target that does not generate verilog.